srf_generation/velocity_model_generation/generate_3d_velocity_model_perturbation.py

```diff
@@ -104,6 +104,8 @@ def generate_velocity_model_perturbation_file_from_config(
         for index, l_p in layer_params.items()
     ]
     try:
+        if n_processes == 1:
```

Comment:
It feels slightly odd to raise an assertion error for the default value for n_processes. I guess there is a good reason for this, and a comment will be helpful.